pandas/tests/arrays/test_integer.py

```diff
@@ -771,6 +771,26 @@ def test_integer_array_constructor_copy():
     assert result._mask is not mask
 
 
+def test_integer_Series_iter_return_native():
+    expected = int
+    result = type(pd.Series([1, 2], dtype="int64").tolist()[0])
+    assert expected == result
```

Comment:
I think it's generally preferred to use `isinstance`, see https://www.python.org/dev/peps/pep-0008/

> Object type comparisons should always use isinstance() instead of comparing types directly.
> 
> Yes: if isinstance(obj, int):
> 
> No:  if type(obj) is type(1):

Reply:
But that would become `isinstance(int, int)` which returns False. 
Are you asking to do it some other way?

Reply:
What I meant was
```
isinstance(pd.Series([1, 2], dtype="int64").tolist()[0], int)
```

Reply:
Oh, okay. I will commit that soon.

Reply:
I have done that. Can you review it, please?